products/pdt/cd/base/cronjob.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: pdt-cronjob
+  namespace: pdt
+  labels:
+    "app.kubernetes.io/name": "pdt-cronjob"
+spec:
+  schedule: "0 0 * * *"
+  selector:
+    matchLabels:
+      "app.kubernetes.io/name": "pdt-cronjob"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: default
```

Comment:
Are you binding the GCP IAM service account to the default service account of the pdt namespace?

Reply:
no, this is just a placeholder. There is a patch on staging that will replace it with `pdt`